regression/strings-smoke-tests/java_last_index_of2/test_last_index_of2.java

```diff
@@ -0,0 +1,8 @@
+public class test_last_index_of2
+{
+   public static void main(/*String[] args*/)
```

Comment:
You can uncomment or remove it. We commented it in the original examples because in the early days of the string support, adding an array of strings to the program would make CBMC much slower, but this should not have any influence now.